### PR TITLE
Enhance PT Client API examples

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Use consistent data path DATASET_PATH
- Make model_class_path configurable

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/hello-world/ml-to-fl/pt/README.md [hunk 1]
```diff
@@ -171,34 +172,15 @@ The modified code can be found in [./code/cifar10_lightning_fl.py](./code/cifar1
 Then we can create the job using sag_pt template:
 
 ```bash
-nvflare job create -force -j ./jobs/lightning -w sag_pt -sd ./code/ -f config_fed_client.conf app_script=cifar10_lightning_fl.py
+nvflare job create -force -j ./jobs/lightning -w sag_pt -sd ./code/ \
+    -f config_fed_client.conf app_script=cifar10_lightning_fl.py \
+    -f config_fed_server.conf key_metric=val_acc_epoch model_class_path="lit_net.LitNet"
```
Comment (SYangster) [inline examples/hello-world/ml-to-fl/pt/README.md:177]: Small thing, the quotes aren't needed for the -f config key-value pair strings right? We could change for consistency or leave as is

**Review by SYangster — APPROVED** [review]
Looks great! Thanks

File: examples/hello-world/ml-to-fl/pt/README.md (hunk 1 above)
Comment (YuanTingHsieh) [reply examples/hello-world/ml-to-fl/pt/README.md:177]: Good suggestion I will fix this for consistency

Comment (YuanTingHsieh) [reply examples/hello-world/ml-to-fl/pt/README.md:177]: I will fix this in the next PR.

**Pull request merged**